sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -130,7 +126,6 @@ export async function createCustomTestProject(
       },
     });
   await polling(client, importTask);
-  console.log("Project created");
```

Comment:
There are similar calls to console.log that are candidates for removal as well.

Reply:
I'll replace all these console.log calls with Azure logger